client/src/components/infoDrawer/infoDrawer.js

```diff
@@ -0,0 +1,180 @@
+      if (!isUserAnno)
+        nonUserAnnoCategories.push(annoMatrix.fetch("obs", catName));
+      else nonUserAnnoCategories.push(null);
+    }
+    const singleValueCategories = (
+      await Promise.all(nonUserAnnoCategories)
+    ).reduce((acc, categoryData, i) => {
+      const catName = allCategoryNames[i];
+
+      const column = categoryData.icol(0);
+      const colSchema = schema.annotations.obsByName[catName];
+
+      const categorySummary = createCategorySummaryFromDfCol(column, colSchema);
+
+      const { numCategoryValues } = categorySummary;
+      if (numCategoryValues === 1) {
+        acc.set(catName, categorySummary.allCategoryValues[0]);
+      }
+      return acc;
+    }, new Map());
+
+    return { singleValueCategories };
+  };
+
+  handleClick = () => {
+    this.setState((state) => {
+      return { isOpen: !state.isOpen };
+    });
+  };
+
+  handleKeyPress = (e) => {
+    if (e.key === "Enter") {
+      this.handleCategoryClick();
+    }
+  };
+
+  render() {
+    const { isOpen } = this.state;
+    const {
+      position,
+      aboutURL,
+      datasetTitle,
+      title,
+      children,
+      schema,
+    } = this.props;
+
+    return (
+      <div
+        role="menuitem"
+        tabIndex="0"
+        onKeyPress={this.handleKeyPress}
+        onClick={this.handleClick}
+      >
+        {children}
+        <Drawer {...{ isOpen, position, title }}>
+          <Async
+            watchFn={InfoDrawer.watchAsync}
+            promiseFn={this.fetchAsyncProps}
+            watchProps={{ schema }}
+          >
+            <Async.Pending>
+              <InfoFormat skeleton {...{ datasetTitle, aboutURL }} />
+            </Async.Pending>
+            <Async.Rejected>
+              {(error) => {
+                console.error(error);
+                return <span>Failed to load info</span>;
+              }}
+            </Async.Rejected>
+            <Async.Fulfilled>
+              {(asyncProps) => {
+                const { singleValueCategories } = asyncProps;
+                return (
+                  <InfoFormat
+                    {...{ datasetTitle, aboutURL, singleValueCategories }}
+                  />
+                );
+              }}
+            </Async.Fulfilled>
+          </Async>
+        </Drawer>
+      </div>
+    );
+  }
+}
+
+const InfoFormat = ({
+  datasetTitle,
+  singleValueCategories = new Map([
+    ["cellxgene", "cellxgene"],
+    ["cellxgenecellxgene", "cellxgene cellxgene"],
+    ["cellxgene cellxgene", "cellxgene"],
+    ["cell", "cellxgene cellxgene"],
+    ["xgene", "cellxgene"],
+    ["cellcellxgenecellxgene", "cellxgene cellxgene cell"],
+    ["cellxgene cellxgene cell", "cellxgene"],
+    ["cellx", "cellxgene cellxgene"],
+  ]),
+  aboutURL = "thisisabouthtelengthofaurl",
+  skeleton = false,
+}) => {
+  return (
+    <div style={{ margin: 24 }}>
+      <H1 className={skeleton ? Classes.SKELETON : null}>{datasetTitle}</H1>
+      {singleValueCategories.size > 0 ? (
```

Comment:
instead of ternary, we can just use `&&` here!

e.g., 

```ts
{singleValueCategories.size && (
        <>
          <H3 className={skeleton ? Classes.SKELETON : null}>
            Dataset Metadata
          </H3>
          <UL>
            {Array.from(singleValueCategories).map((pair) => {
              return (
                <li
                  className={skeleton ? Classes.SKELETON : null}
                  key={pair[0]}
                >{`${pair[0]}: ${pair[1]}`}</li>
              );
            })}
          </UL>
        </>
      )}
```

Reply:
Added (but kept the greater than comparison for readability)